integration/spark-datasource/src/test/scala/org/apache/spark/sql/carbondata/datasource/SparkCarbonDataSourceTest.scala

```diff
@@ -76,6 +76,12 @@ class SparkCarbonDataSourceTest extends FunSuite with BeforeAndAfterAll {
     spark.sql("drop table if exists testformat")
   }
 
+  test("test alter") {
+//    spark.sql("create table t1(name string, age int) using carbon")
+//    spark.sql("insert into t1 select 'abc',4")
+    spark.sql("alter table t1 add columns(add string)")
```

Comment:
What is the purpose of this testcase in this PR? Please remove the commented code and do the assert check whether it achieves expected result or not.

Reply:
this was by mistake, reverted it